### ISSUE TITLE
Refactor `secondRepoId` for clarity

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#5231]
### Overview
We need to change `secondRepoId` to a more descriptive name so that it is clearer it will contain additional repository identification numbers (second, third, fourth, etc) for a project.

### Details
- In future issue #5160, the code will be modified to support projects with 2 or more repositories. 

#### Developer
- [x] In _projects/civic-tech-index.md [^1], change
  ```
  secondRepoId: '275296439'
  ```
  to
  ```
  additional-repo-ids: '275296439'
  ```
- [x] In _layouts/project.html [^2], change
  ```
  secRepoId="{{ page.secondRepoId }}"
  ```
  to
  ```
  additionalRepoIdNums="{{ page.additional-repo-ids }}"
  ```
- [x] In assets/js/current-projects.js [^3], change the following:
  - [x] Change
    ```
    'secondRepoId': {{project.secondRepoId | default: 0}},
    ```
    to
    ```
    'additionalRepoIds': {{project.additional-repo-ids | default: 0}},
    ```
  - [x] Change
    ```
                if(project.secondRepoId != 0){
                    const secMatchingProject = projectLanguagesArr.find(x=> x.id === project.secondRepoId);
                    const langArr = [...matchingProject.languages, ...secMatchingProject.languages];
    ```
    to
    ```
                if(project.additionalRepoIds != 0){
                    const additionalMatchingProject = projectLanguagesArr.find(x=> x.id === project.additionalRepoIds);
                    const langArr = [...matchingProject.languages, ...additionalMatchingProject.languages];
    ```
- [x] In assets/js/project.js [^4], change
  ```
  if (scriptTag.getAttribute("secRepoId")){
      const secRepoId = scriptTag.getAttribute("secRepoId");    
      const firstLangs = project.languages;
      const secLangs = findProjectById(secRepoId).languages;
  
      let languagesArr = [...firstLangs, ...secLangs];
  ```
  to
  ```
  if (scriptTag.getAttribute("additionalRepoIdNums")){
      const additionalRepoIdNums = scriptTag.getAttribute("additionalRepoIdNums");    
      const firstLangs = project.languages;
      const additionalLangs = findProjectById(additionalRepoIdNums).languages;
  
      let languagesArr = [...firstLangs, ...additionalLangs];
  ```
- [x] Using Docker, on the following web pages, check everything remains the same as on the live website:
  - [x] Projects page: https://www.hackforla.org/projects/, including checking the functionality of the filters and the contents of the Civic Tech Index card
  - [x] Civic Tech Index page: https://www.hackforla.org/projects/civic-tech-index, including checking the Languages field

#### Merge Team
- [x] Once the pull request associated with this issue is merged, go to #5160 and check if all the issues listed under the Dependencies section are closed. If the dependencies are all closed, move #5160 into the New Issue Approval column, remove the "Dependency" label on it and add a "ready for prioritization" label to it.

### Resources/Instructions
[^1]: [_projects/civic-tech-index.md](https://github.com/hackforla/website/blob/gh-pages/_projects/civic-tech-index.md?plain=1)
[^2]: [_layouts/project.html](https://github.com/hackforla/website/blob/gh-pages/_layouts/project.html)
[^3]: [assets/js/current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js)
[^4]: [assets/js/project.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/project.js)

**Comment by davidwiese** [comment]
ETA/Availability: 8/31 by 4pm PST

**Comment by t-will-gillis** [comment]
Hi @davidwiese - Appreciate your enthusiasm! However, we ask that you are working on/ have open only one issue at a time. Unfortunately, we need to unassign you from this one until your other issue is closed/ PR is merged.

**Comment by davidwiese** [comment]
Availability: 8/31 until 4pm PST
ETA: 9/1 4pm PST

**Comment by davidwiese** [comment]
While working on this issue, I noticed that there is a typographical inconsistency on both the Civic Tech Index project card, as well as its actual page (https://www.hackforla.org/projects/civic-tech-index). It lists "Google docs" as one of the tools, but when that tool is listed on any other project card or page, it is "Google Docs" with the capitalized "D". I didn't want to make a change to this because it was outside of the scope of this issue. Is this something I can attempt to create an issue for? Or would it be an emergent request? If someone can point me in the right direction, I would like to take a shot at it.

**Comment by t-will-gillis** [comment]
Hi @davidwiese Thanks for noticing and asking the question about the text. This would be perfect for a new issue; however, in this specific case, an issue already exists: #5341. 

But we definitely will welcome and encourage any help with writing issues! If you are interested, there are some outstanding ERs in the queue- that is, ERs that need to have issues written, for example #5098 and #4564. If you are interested in either of these or another one, let us know and we can talk briefly about it and make sure we're all on the same page. 

Here is a link from the wiki that might be useful... https://github.com/hackforla/website/wiki/How-to-create-issues
